packages/amqp-client/amqp-client.1.0.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Anders Fugmann <anders@fugmann.net>"
+authors: [
+  "Anders Fugmann"
+]
+homepage: "https://github.com/andersfugmann/amqp-client"
+bug-reports: "https://github.com/andersfugmann/amqp-client/issues"
+license: "BSD3"
+dev-repo: "https://github.com/andersfugmann/amqp-client.git"
+build: [ ]
+install: [make "install"]
+remove: [make "uninstall"]
+depends: [
+  "ocamlfind" {build}
+  "omake" {build}
+  "xml-light" {build}
```

Comment:
Build time only. Its only used to parse the amqp specification when generating the autogenerated protocol description.